src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,34 @@
+import { NgModule} from '@angular/core';
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import {
+    MatButtonModule, MatCardModule, MatIconModule, MatInputModule, MatMenuModule,
+    MatProgressSpinnerModule
+} from '@angular/material';
+
+@NgModule({
+    imports: [
+        FlexLayoutModule,
+        FormsModule,
+        MatButtonModule,
+        MatCardModule,
+        MatIconModule,
+        MatInputModule,
+        MatProgressSpinnerModule,
+        MatMenuModule,
+        ReactiveFormsModule
+    ],
+    exports: [
+        FlexLayoutModule,
+        FormsModule,
+        MatButtonModule,
+        MatCardModule,
+        MatIconModule,
+        MatInputModule,
+        MatProgressSpinnerModule,
+        MatMenuModule,
+        ReactiveFormsModule
```

Comment:
don't export modules - there is no need to do it - if you will import a SharedModule it will make all imported submodules available